core/templates/dev/head/pages/exploration_editor/simple_editor_tab/StatesToQuestionsService.js

```diff
@@ -160,6 +160,15 @@ oppia.factory('StatesToQuestionsService', [
         // EndExploration.
         var state = SimpleEditorShimService.getState(missingStateNames[0]);
         var interactionId = state.interaction.id;
+        SimpleEditorShimService.saveInteractionId(
+              currentStateName, DEFAULT_INTERACTION.ID);
```

Comment:
Continuation lines should be indented by 2 from the previous one, not 4. Ditto below.